archetypes/archetypes/src/main/archetype/mp/custom/database-outputs.xml

```diff
@@ -332,9 +332,6 @@ docker run --rm --name mysql -p 3306:3306 \
                       target="src/main/resources/META-INF/init_script.sql"/>
                 <templates engine="mustache" transformations="mustache,packaged">
```

Comment:
Please remove the whole `templates` block as it contained only the file you deleted.

Reply:
Fixed.